front/src/containers/Profile/style.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components'
+import { StyleSheet } from 'react-native'
+
+export const GlobalStyle = StyleSheet.create({
+  darkThemeSwitch: {
+    marginLeft: 'auto',
+    transform: [{ scaleX: 1.2 }, { scaleY: 1.2 }]
```

Comment:
oq faz isso ? ( **transform: [{ scaleX: 1.2 }, { scaleY: 1.2 }]** )

Reply:
This increases the size of switch component
https://developer.mozilla.org/pt-BR/docs/Web/CSS/transform-function/scale